doc/usage/extensions/autosummary.rst

```diff
@@ -201,6 +201,25 @@ also use these config values:
 
    .. versionadded:: 2.1
 
+   .. versionchanged:: 4.3
+
+      If ``autosummary_ignore___all__`` is ``False``, this configuration value
+      is ignored for members listed in ``__all__``.
+
+.. confval:: autosummary_ignore___all__
```

Comment:
In autodoc extension, this option is called as `:ignore-module-all:`. So it would be better to follow its name.
https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-option-automodule-members